src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,67 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.ValidationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int VALID_LOGIN_LENGTH = 6;
+    private static final int VALID_PASSWORD_LENGTH = 6;
+    private static final int VALID_AGE = 18;
+    private static final long ID = 111232;
     private final StorageDao storageDao = new StorageDaoImpl();
 
-    @Override
     public User register(User user) {
-        return null;
+        validateUser(user);
+        storageDao.add(user);
+        return user;
+    }
+
+    private void validateUser(User user) {
+        isLoginExists(user);
+        isLoginNull(user);
+        isPasswordNull(user);
+        isPasswordInvalid(user);
+        isLoginInvalid(user);
+        isAgeInvalid(user);
+    }
+
+    private void isLoginInvalid(User user) {
```

Comment:
Better to rename to 'validateLogin' or 'checkLogin', because word 'is' means that boolean variable will be returned